kiwi_lint/model_form.py

```diff
@@ -0,0 +1,23 @@
+from pylint import interfaces
+from pylint import checkers
+from pylint_django.utils import node_is_subclass
+
+
+class ModelFormChecker(checkers.BaseChecker):
```

Comment:
Must be moved to forms.py from #1742 and the msg ID number updated afterwards. Keep the implementation in a separate class.

Reply:
@atodorov, I'll update the ID after #1742 is merged